specs/linux/docker_container_deb_packages.table

```diff
@@ -0,0 +1,16 @@
+table_name("docker_container_deb_packages")
```

Comment:
Its probably a non-goal to support other container runtimes (which would shorten this to `container_deb_packages`), but I wonder if there's an alternative name that is perhaps a little less verbose? `docker_debs`? Is there a plan to support non-debs? Perhaps `docker_container_packages`? I'm just spitballing obviously, I'm not opposed to this name, but I think this kind of data would be useful with other container runtimes, other package formats, etc.

Reply:
@marpaia Yeah right now not a goal to add other container runtimes, but if we keep it generic then we will have to force users to add additional Where clauses like `WHERE docker_container_id=2121` or `WHERE lxc_container_name=foo`. Totally depends on what user experience we want from this.

Reply:
My 2-cents would be `docker_deb_packages` or `container_deb_packages`, as this type of table (concept at least) could be expanded to support either 1.) Other types of packages in docker containers, or 2.) other types of containers and their subsequent packages.

Reply:
I would personally vote for `container_deb_packages` out of the two so that it's more vendor neutral and welcoming to future contributors who may want to help by contributing support for other container runtimes.